client/lib/post-normalizer/rule-content-link-jetpack-carousels.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External Dependencies
+ */
+import { forEach, get } from 'lodash';
+
+export default function linkJetpackCarousels( post, dom ) {
```

Comment:
Also, for newbs like me, can we include that all galleries (including wpcom) are considered jetpack galleries?